keyboards/ymdk/ymd60/keymaps/via/rules.mk

```diff
@@ -0,0 +1,2 @@
+VIA_ENABLE = yes
+
```

Comment:
Usually `via` keymaps for ATmega32u4-based boards also have `LTO_ENABLE = yes` here; depending on how close the firmware size is to the limit, you may need to add it here.